components/dashboard/src/teams/GitIntegrationsPage.tsx

```diff
@@ -24,11 +24,14 @@ export default function GitAuth() {
 export const OrgSettingsPageWrapper: FunctionComponent = ({ children }) => {
     const currentOrg = useCurrentOrg();
 
+    const title = "Git Integrations";
```

Comment:
🙏🏻  could we rename it to `Git Auth` to emphasize that this feature is about authorization of Git commands (and APIs). AFAIR the _integrations_ was just some middle ground as we had to incorporate the login thing into it, but here we can clarify on the expectations.